packages/treasury/test/test-interest.js

```diff
@@ -10,6 +10,9 @@ import { makeInterestCalculator, SECONDS_PER_YEAR } from '../src/interest';
 const ONE_DAY = 60n * 60n * 24n;
 const ONE_MONTH = ONE_DAY * 30n;
 const ONE_YEAR = ONE_MONTH * 12n;
+const BASIS_POINTS = 10000n;
+const HUNDRED_THOUSAND = 100000n;
```

Comment:
BASIS_POINTS makes sense. The mere numeric ones are not clarifying as to their purpose. TEST_SCALE to make sure we have significant digits?

Reply:
Ah. I see below that TEN_MILLION is "LARGE_DEBT"? Perhaps HUNDRED_THOUSAND is MEDIUM_DEBT?